lualib/resty/events/worker.lua

```diff
@@ -321,7 +332,7 @@ local function post_event(self, source, event, data, spec)
     EVENT_T.source = source
     EVENT_T.event = event
     EVENT_T.data = data
-    EVENT_T.wid = _worker_id
+    EVENT_T.wid = self._worker_id or ngx_worker_id() or -1
```

Comment:
Is the `or` necessary since it's initialized in `:init`?

Reply:
Some tests seem to publish events before they run `:init` so that's why it is there.